oteapi/strategies/resource/application_vnd_postgres.py

```diff
@@ -0,0 +1,115 @@
+"""Strategy class for application/vnd.postgresql"""
+# pylint: disable=unused-argument
+from typing import TYPE_CHECKING, Optional
+
+import psycopg
+from pydantic import Field
+from pydantic.dataclasses import dataclass
+
+from oteapi.models import AttrDict, DataCacheConfig, ResourceConfig, SessionUpdate
+from oteapi.models.resourceconfig import HostlessAnyUrl
+
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import Any, Dict
+
+
+class PostgresConnectionConfig(AttrDict):
+    """Configuration data model for
+    [`PostgresResourceStrategy`][oteapi.strategies.resource.application_vnd_postgres.PostgresParseStrategy]."""
+
+    connection_dict: dict = Field(
```

Comment:
Instead of having a completely generic connection field, it would be better (and more intuitive for the users) if you define all the necessary fields explicitly. e.g. user, password and database